### ISSUE TITLE
Print sparky and sparky web logs in web UI

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#18]
Can be put behind admin auth after #17 is implemented

**Comment by melezhik** [comment]
In most of the time there is no sensitive information in sparkyd, sparky-web logs, why not make this public available ?

**Comment by thibaultduponchelle** [comment]
I think there is the config hash dump that could be sensible (not sure)

**Comment by melezhik** [comment]
yeah, it could ... for simplicity I'd start with public facing , and with an option to disable it  through a sparky global config

**Comment by thibaultduponchelle** [comment]
Great 👍
